okta/plugin.spec.yaml

```
@@ -131,6 +164,33 @@ connection:
     description: Okta key
     type: credential_secret_key
     required: true
+triggers:
+  users_added_removed_from_group:
+    title: Users Added Removed from Group
```

```suggestion
    title: Users Added Removed from Group
```
What about something like "Monitor User Groups"?